### DISCUSSION TITLE
Why are public player sources not on github?

### DISCUSSION THREAD

**Discussion by krono-i2** [AzuraCast/AzuraCast#6945] [Q&A]
Hello!
I realized that the files `web` folder link others in `vendor`, but the latter is empty on github.
Where can I find all the sources of the public player?
Thanks a lot.

Ivan

**Comment by BusterNeece** [comment]
Here's the file you're looking for: https://github.com/AzuraCast/AzuraCast/blob/a22a8a45cdbbc8bd4d51bc66cdcaa9dabceb7972/frontend/src/components/Public/FullPlayer.vue

We build our Vue components down into static JS/CSS assets which are included in our AzuraCast images. That's the raw source for our player page.
